teslajsonpy/car.py

```diff
@@ -485,7 +486,7 @@ def software_update(self) -> dict:
     @property
     def steering_wheel_heater(self) -> bool:
         """Return steering wheel heater option."""
-        return self._vehicle_data.get("climate_state", {}).get("steering_wheel_heater")
+        return self._vehicle_data.get("climate_state", {}).get("steering_wheel_heater") is not None
```

Comment:
Does this mean the steering_wheel may not have an unavailable state? I realize the typing says `bool`, but if we need to convey unavailability the fix is to make it `bool | None`

Reply:
`steering_wheel_heater` is only used for "is it available"; and `is_steering_wheel_heater_on` is used for on/off determination.

I thought about renaming this property to `steering_wheel_heater_available` but that would break backwards compatibility with the tesla integration, in case that matters.

Reply:
Ok that's fine. I didn't realize it was just feature availability.